pkg/apis/rhjmc/v1beta1/containerjfr_types.go

```diff
@@ -43,7 +43,8 @@ import (
 // ContainerJFRSpec defines the desired state of ContainerJFR
 // +k8s:openapi-gen=true
 type ContainerJFRSpec struct {
-	Minimal bool `json:"minimal"`
+	Minimal bool                `json:"minimal"`
+	Secrets []CertificateSecret `json:"secrets"`
```

Comment:
Let's call this `TrustedCertSecrets` to be more specific about the intended use. We should also add a comment here to document what the secrets are for.